project_recalculate/models/resource_calendar.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# See README.rst file on addon root folder for license details
+
+from openerp import models, api
+from datetime import datetime, timedelta
+
+
+class ResourceCalendar(models.Model):
+    _inherit = 'resource.calendar'
+
+    @api.v7
```

Comment:
Because I inherit from models.Model but I need to call to this method in old api flavor.